src/org/gk/elv/InstanceZoomablePathwayEditor.java

```diff
@@ -63,17 +63,14 @@
 import org.gk.render.FlowLine;
 import org.gk.render.HyperEdge;
 import org.gk.render.Node;
-import org.gk.render.NodeAttachment;
 import org.gk.render.Note;
 import org.gk.render.ProcessNode;
 import org.gk.render.RenderUtility;
 import org.gk.render.Renderable;
 import org.gk.render.RenderableComplex;
-import org.gk.render.RenderableFeature;
 import org.gk.render.RenderablePathway;
 import org.gk.render.RenderableReaction;
 import org.gk.render.RenderableRegistry;
-import org.gk.schema.GKSchemaClass;
 import org.gk.schema.SchemaClass;
 import org.gk.util.DialogControlPane;
 
```

Comment:
This is a class you should not touch actually. Please focus on what you need to do.

Reply:
Ah, autoformatter mistake, sorry! I've changed my eclipse config to stop that behavior, should I reset the above change?